en_ewt-ud-train.conllu

```diff
@@ -152239,7 +152239,7 @@
 13	and	and	CCONJ	CC	_	16	cc	16:cc	_
 14	the	the	DET	DT	Definite=Def|PronType=Art	16	det	16:det	_
 15	where	where	NOUN	NN	Number=Sing	16	compound	16:compound	_
-16	to's	to'	NOUN	NNS	Number=Plur	12	conj	9:obj|12:conj:and	SpaceAfter=No
+16	to's	to	NOUN	NNS	Number=Plur|Typo=Yes	12	conj	9:obj|12:conj:and	CorrectForm=tos|SpaceAfter=No
```

Comment:
Since "to" is not normally a noun and "tos" looks like it would be pronounced differently I suspect this is commonly pluralized with the apostrophe even in edited text.

Reply:
Are you saying in terms of the Typo=Yes?  I'm not sure how I feel about that, but I can remove it for now

Reply:
Yeah let's not call it a typo

Reply:
fixed.  other than that, good to go?

Reply:
Yeah once you push that I'll merge